app/controllers/idv/review_controller.rb

```diff
@@ -52,6 +53,18 @@ def create
 
     private
 
+    def step_indicator_steps
+      steps = Idv::Flows::DocAuthFlow::STEP_INDICATOR_STEPS
+      return steps if idv_session.address_verification_mechanism != 'gpo'
+      steps.map do |step|
+        if step[:name] == :verify_phone_or_address
+          step.merge({ status: :pending })
```

Comment:
big fan of the change! small ruby golf moment, curly braces are optional if the last arg is a hash

```suggestion
          step.merge(status: :pending)
```

Reply:
> big fan of the change! small ruby golf moment, curly braces are optional if the last arg is a hash

Oops, good catch!